### PR TITLE
Add vscode config

### PR DESCRIPTION
**Pull request by dels07**
Add `major/minor/patch` configuration when comparing package(s)

### CODE REVIEW

File: src/diagnostics/getPackageInfo.ts [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+import { workspace } from 'vscode';
 import { fetchPackage } from "../utils/packages"
 
 export async function getPackageInfo([name, version]: string[]) {
   const info = await fetchPackage(name)
   const localVersion = semverCoerce(version)
+  const versionToCompare = workspace.getConfiguration().get<string>('npm-outdated.version')
+  const versionDiff = info && localVersion && semverDiff(info.version, localVersion)
+
+  const versionMap: any = {
```
Comment (mskelton) [inline src/diagnostics/getPackageInfo.ts:12]:
```suggestion
  const versionMap: Record<string, number> = {
```

File: package.json [hunk 2]
```diff
@@ -64,5 +64,23 @@
   },
   "dependencies": {
     "semver": "^7.3.2"
+  },
+  "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "npm Outdated",
+      "properties": {
+        "npm-outdated.version": {
```
Comment (mskelton) [inline package.json:73]: `version` doesn't feel right to me, what about something like range or level?

```suggestion
        "npm-outdated.range": {
```

File: package.json [hunk 3]
```diff
@@ -64,5 +64,23 @@
...
+          "type": "string",
+          "default": "patch",
+          "enum": [
+            "major",
+            "minor",
+            "patch"
+          ],
+          "markdownDescription": "Specify which minimum version of npm packages to compare. See the [Semver web](https://semver.org/) for more information."
```
Comment (mskelton) [inline package.json:81]:
```suggestion
          "markdownDescription": "Minimum semver bump required for a package to display as outdated. See npm's [Semver docs](https://docs.npmjs.com/cli/v6/using-npm/semver) for more details."
```

File: src/diagnostics/getPackageInfo.ts (hunk 1 above)
Comment (dels07) [reply src/diagnostics/getPackageInfo.ts:12]: Thanks for correct type-hint

File: package.json (hunk 2 above)
Comment (dels07) [reply package.json:73]: `level` might sound appropriate since range usually denotes value from min to max

**Comment by dels07** [comment]
PR updated, thanks

**Review by mskelton — APPROVED** [review]
Nice!

**Pull request merged**